lib/administrate/namespace.rb

```diff
@@ -14,6 +14,16 @@ def resources
 
     attr_reader :namespace
 
+    def whitelisted_controller_paths
+      if dashboard_manifest_missing?
+        return namespace_controller_paths.uniq
+      end
+
+      namespace_controller_paths.uniq.select do |path|
+        dashboard_paths.include? path
+      end
+    end
```

Comment:
I could have gone either way here - guard clause or if/else. If everyone prefers the more explicit if/else...? Then I'm good with that. Just let me know.